graylog2-web-interface/src/views/bindings.jsx

```diff
@@ -344,4 +349,7 @@ export default {
   'views.hooks.loadingView': [
     requirementsProvided,
   ],
+  'views.elements.header': [() => hasLegacyFieldCharts() && (
+    <MigrateFieldCharts />
+  )],
```

Comment:
I want to display the `<MigrateFieldCharts />` inside the `<HeaderElements />` component and thought it is enough to add this section, but it never gets executed. Did I forgot something?

Reply:
Thanks to @kmerz I figured out the `HeaderElements` component gets called before we are registering the `ViewBindings` inside the `index.jsx`, it looks like it's caused by the script import order. I am searching for a solution.

Reply:
Answer: Not caused by script order, but the way we were importing the ExtendedSearchPage